lib/vmdb/loggers/request_logger.rb

```diff
@@ -0,0 +1,38 @@
+module Vmdb::Loggers
+  class RequestLogger < ManageIQ::Loggers::Base
+    def log_to(evm, message, severity)
+      if evm
+        $log && $log.method(severity).call(message)
+      else
+        $miq_ae_logger && $miq_ae_logger.method(severity).call(message)
+      end
```

Comment:
Minor, but this is simpler:

```suggestion
      if evm
        $log && $log.public_send(severity, message)
      else
        $miq_ae_logger && $miq_ae_logger.public_send(severity, message)
      end
```